diff --git a/dist/base.js b/dist/base.js
index 3ac40c4d8163c09a885cbeb25320612181476fc0..77656c13a2ab60a9dcf2bacb1425c279394510c5 100644
--- a/dist/base.js
+++ b/dist/base.js
@@ -1,9 +1,8 @@
-import { pkg } from './utils/package.js';
 import { LastFMRequest } from './request.js';
 export default class Base {
     key;
     userAgent;
-    constructor(key, userAgent = `simple-fm v${pkg.version} - a simple Last.fm wrapper written in TypeScript (https://github.com/solelychloe/simple-fm)`) {
+    constructor(key, userAgent = `simple-fm - a simple Last.fm wrapper written in TypeScript (https://github.com/solelychloe/simple-fm)`) {
         this.key = key;
         this.userAgent = userAgent;
     }
diff --git a/dist/index.js b/dist/index.js
index 06b05c309a5516a299c07fe9c896265b1b10ed62..705486bbdb64f2efd7fb1a104719ec1d921cfe26 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -6,7 +6,6 @@ import Tag from './classes/tag.class.js';
 import Track from './classes/track.class.js';
 import User from './classes/user.class.js';
 import LastFMError from './utils/error.js';
-import { pkg } from './utils/package.js';
 export default class SimpleFMClient {
     key;
     album;
@@ -19,7 +18,7 @@ export default class SimpleFMClient {
     constructor(key, options = {}) {
         this.key = key;
         this.validateApiKey();
-        options.userAgent ??= `simple-fm v${pkg.version} - a simple Last.fm wrapper written in TypeScript (https://github.com/solelychloe/simple-fm)`;
+        options.userAgent ??= `simple-fm - a simple Last.fm wrapper written in TypeScript (https://github.com/solelychloe/simple-fm)`;
         this.album = new Album(key, options.userAgent);
         this.artist = new Artist(key, options.userAgent);
         this.chart = new Chart(key, options.userAgent);
diff --git a/dist/utils/package.d.ts b/dist/utils/package.d.ts
deleted file mode 100644
index 7b3f89906f64f4be24b8d3f27de119614410c26f..0000000000000000000000000000000000000000
diff --git a/dist/utils/package.js b/dist/utils/package.js
deleted file mode 100644
index aa9893c201faa50afc1951e67ccf432f8c54918a..0000000000000000000000000000000000000000
